voltdbclient/api.go

```diff
@@ -0,0 +1,243 @@
+/* This file is part of VoltDB.
```

Comment:
Can the filenames be better like voltdbclient.go if its client. same with Connection Conn things could be more explicit right? Feel free to rename what makes sense from what you got from old repo.